sdks/java/core/src/main/java/org/apache/beam/sdk/util/WindowingStrategies.java

```diff
@@ -122,8 +126,13 @@ public static ClosingBehavior fromProto(RunnerApi.ClosingBehavior proto) {
   }
 
   // This URN says that the coder is just a UDF blob the indicated SDK understands
+  // TODO: standardize such things
   private static final String CUSTOM_CODER_URN = "urn:beam:coders:custom:1.0";
```

Comment:
Nit: should we be using 1.0 before beam is 1.0 (and the TODO about standardization is resolved)?